apps/trip_plan/lib/trip_plan/personal_detail.ex

```diff
@@ -64,4 +67,13 @@ defmodule TripPlan.PersonalDetail.Step do
   def human_relative_direction(:elevator), do: "Take the elevator"
   def human_relative_direction(:uturn_left), do: "Make a U-turn"
   def human_relative_direction(:uturn_right), do: "Make a U-turn"
+  def human_relative_direction(:enter_station), do: "Enter the station"
+  def human_relative_direction(:exit_station), do: "Exit the station"
+  def human_relative_direction(:follow_signs), do: "Follow signs"
+
+  @spec human_relative_preposition(relative_direction) :: binary
```

Comment:
what a neat addition!

Reply:
I didn't even realize this had been implemented until the json started failing to parse because there were unexpected values! I think it's a quite new feature, from within the last couple weeks. I hope they make grammatical sense in most cases, I'm not certain exactly where the location strings are being pulled from, I assume it's GTFS pathways? And I really don't like when it does the thing like in your screenshot where it says to follow the same signs multiple times in a row... I'll try bringing that up with them, but we might need to filter out consecutive dupes on our side.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/issues/5322
Made an issue in the OTP repo on the dupes